lib/componentRelay.ts

```diff
@@ -131,16 +131,16 @@ export default class ComponentRelay {
     if (initialPermissions && initialPermissions.length > 0) {
       this.initialPermissions = initialPermissions
     }
-    if (options?.coallesedSaving) {
+    if (typeof options?.coallesedSaving === 'boolean') {
       this.coallesedSaving = options.coallesedSaving
     }
-    if (options?.coallesedSavingDelay) {
+    if (typeof options?.coallesedSavingDelay === 'number') {
       this.coallesedSavingDelay = options.coallesedSavingDelay
     }
-    if (options?.acceptsThemes) {
-      this.component.acceptsThemes = options?.acceptsThemes ?? true
+    if (typeof options?.acceptsThemes === 'boolean') {
```

Comment:
Hmm..are you trying to check for null or undefined? I would do that instead. Maybe create a helper function.

Reply:
See https://github.com/standardnotes/component-relay/pull/41/commits/e06a73daf06f3478414861fff5deec2eb32f5568